github/orgs_outside_collaborators_test.go

```diff
@@ -122,7 +126,9 @@ func TestOrganizationsService_ConvertMemberToOutsideCollaborator_NonMemberOrLast
 	mux.HandleFunc("/orgs/o/outside_collaborators/u", handler)
 
 	_, err := client.Organizations.ConvertMemberToOutsideCollaborator(context.Background(), "o", "u")
-	if err, ok := err.(*ErrorResponse); !ok || err.Response.StatusCode != http.StatusForbidden {
-		t.Errorf("Organizations.ConvertMemberToOutsideCollaborator_NonMember returned error: %v", err)
+	if err, ok := err.(*ErrorResponse); !ok {
+		t.Errorf("Organizations.ConvertMemberToOutsideCollaborator did not return an error")
+	} else if err.Response.StatusCode != http.StatusForbidden {
+		t.Errorf("Organizations.ConvertMemberToOutsideCollaborator did not return 403 Status code")
```

Comment:
Nit; capital "S" in "Status code" here but not anyone else. It should be lowercase for consistency.